headers/io/bgwriter.h

```diff
@@ -0,0 +1,16 @@
+//
+// Created by Sameer Raj on 05/08/24.
+//
+
+#ifndef BGWRITER_H
+#define BGWRITER_H
+
+#include "../storage/buffer.h"
+#include "../storage/page.h"
+
+namespace io {
+    void startBgWriter(const BufferDescriptors& bufferDescriptors,
+       BufferId victimBuffer,
```

Comment:
From the implementation this can be const as well

Reply:
Got this in CLion: Clang-Tidy: Parameter 'victimBuffer' is const-qualified in the function declaration; const-qualification of parameters only has an effect in function definitions

when added const BufferId victimBuffer